pkg/asset/agent/joiner/addnodesconfig.go

```diff
@@ -0,0 +1,79 @@
+package joiner
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/pkg/errors"
+)
+
+const (
+	addNodesParamsFile = ".addnodesparams"
```

Comment:
Is there a reason why this file is a . file?

Reply:
This is a technical file used to propagate the command line parameters into the assets graph, so it's not expected to be modified by the user (in a similar way to the `.openshift_install_*` files)